lint-staged.config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+const micromatch = require( 'micromatch' );
+
+/**
+ * Internal dependencies
+ */
+const { plugins } = require( './plugins.json' );
+
+module.exports = {
+	'**/*.js': ( files ) =>
+		`npm run lint-js ${ files.length > 10 ? '' : files.join( ' ' ) }`,
```

Comment:
It's just to run linting on whole codebase in case there are multiple files which are staged.

Reply:
If modifying more than 10 files though, perhaps you don't want to check _all_ the files? Why not just pass the files always?

Reply:
My initial thought was to run linting on the whole codebase if changes are big. In big changes, sometimes a few things are missed. Let's say you removed some depeendecy from `package.json` but they are still in some unedited file. ESlint have a rule to check that but such things can also be caught on bundling or runtime.

Though I am not fully sure if we need to keep this but if changes are big it can make sense to run linting on whole codebase?

Reply:
I'd say to remove the `files.length` check. The 10 seems a bit arbitrary. In any case, the GHA workflow will check against the entire codebase anyway.

Reply:
Humm makes sense. Reverting it.